resources/fixtures/Briteq/Briteq-BT-575S.qxf

```diff
@@ -166,29 +166,14 @@
   <Channel Number="6">No function</Channel>
   <Channel Number="7">Gobo 1</Channel>
   <Channel Number="8">Gobo 2</Channel>
-  <Channel Number="9">Gobo rotation</Channel>
+  <Channel Number="9" ActsOn="8">Gobo 2 Rotation</Channel>
   <Channel Number="10">Prism</Channel>
```

Comment:
What happened here?? An entire mode went away

Reply:
No. The name of the channel was changed (line 70) from "Gobo rotation" to "Gobo 2 Rotation" in order to make it clearer, which Gobo group rotates.

Reply:
Please look closely at the diff.
"8 Channel" mode disappeared and channels from 11 to 15 in "16 channel" mode disappeared.
Basically this definition is now broken.

Reply:
Apologies! We will correct it and check for errors like this when preparing future pull requests.